Makefile

```diff
@@ -6,7 +6,7 @@ build:
 	cargo build -Z unstable-options --release --out-dir ./bin
 
 test:
-	cargo test --release --workspace --exclude ipc_e2e
+	cargo test --release --workspace --exclude ipc_e2e itest
```

Comment:
If we excluse `itest` let's add a new rule called `make itests` that runs integration tests. WDYT?

Reply:
added